hummingbot/client/command/previous_strategy_command.py

```diff
@@ -0,0 +1,81 @@
+from hummingbot.core.utils.async_utils import safe_ensure_future
+from hummingbot.client.config.config_validators import validate_bool
+from hummingbot.client.config.global_config_map import global_config_map
+from hummingbot.client.config.config_helpers import (
+    parse_cvar_value,
+    parse_config_default_to_text,
+)
+from .import_command import ImportCommand
+from hummingbot.client.config.config_var import ConfigVar
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from hummingbot.client.hummingbot_application import HummingbotApplication
+
+
+class PreviousCommand:
+    def previous_statrategy(
+        self,  # type: HummingbotApplication
+        option: str,
+    ):
+        if option is not None:
+            pass
+
+        previous_strategy_file = global_config_map["previous_strategy"].value
+
+        if previous_strategy_file is not None:
+            safe_ensure_future(self.prompt_for_previous_strategy(previous_strategy_file))
+        else:
+            self.notify("No previous strategy found.")
+
+    async def prompt_for_previous_strategy(
+        self,  # type: HummingbotApplication
+        file_name,
+    ):
+        self.app.clear_input()
+        self.placeholder_mode = True
+        self.app.hide_input = True
+
+        previous_strategy = ConfigVar(
+            key="previous_strategy_answer",
+            prompt=f"Dou you want to import the previously stored strategy? ({file_name}) (Yes/No) >>>",
```

Comment:
the prompt issue is due to the use of the word strategy as the completer in hummingbot/client/ui/completer.py looks for strategy in the prompt first. One possible method is to change to previously stored config to resolve this. but up to you.

Reply:
also typo Dou -> Do